config/locales/candidate_interface/restructured_work_history.yml

```diff
@@ -21,3 +49,26 @@ en:
               blank: Select whether you have any work history
             explanation:
               blank: Please explain why you’ve been out of the workplace
+        candidate_interface/restructured_work_history/job_form:
+          attributes:
+            role:
+              blank: Enter your job title
+              too_long: Job title must be %{count} characters or fewer
+            organisation:
+              blank: Enter the name of your employer
+              too_long: Name of the employer must be %{count} characters or fewer
+            working_with_children:
+              blank: Select if this job involves working in a school or with children
+            commitment:
+              blank: Select if this job is full time or part time
+            start_date:
+              invalid: Enter a start date in the correct format
+              before: Enter a start date that is before the end date
+              in_the_future: Enter a start date that is not in the future
+            currently_working:
+              blank: Select if you are still working in this job
+            end_date:
+              invalid: Enter an end date in the correct format
+              in_the_future: Enter an end date that is not in the future
+            relevant_skills:
+              blank: Select if you use skills relevant to teaching in this job
```

Comment:
The error messaging for the bottom two fields are slightly off 

![image](https://user-images.githubusercontent.com/42515961/106361324-40597680-6315-11eb-896b-5c7d6291c2ab.png)

I think the radios also need to be validated for presence.

`  validates :role,
                :organisation,
                :commitment,
                :relevant_skills,
                :currently_working,
                presence: true`

Reply:
Ah my bad. Have fixed this now